### PR TITLE
docs(tutorial): ss2 instructions

### PR DESCRIPTION
**Pull request by WJaworskiRobotec**
## Description

Instructions on running scenario_simulator_v2 with autoware universe

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [X] I've confirmed the [contribution guidelines].
- [X] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/tutorials/scenario-simulation/.pages
Comment (kenji-miyake) [inline docs/tutorials/scenario-simulation/.pages:5]: @WJaworskiRobotec Could you add files under the `planning-simulation/` directory, and keep `rosbag-replay-simulation.md` and `digital-twin-simulation.md` as-is?

File: docs/tutorials/scenario-simulation/installation.md
Comment (kenji-miyake) [inline docs/tutorials/scenario-simulation/installation.md:9]: I think it's better to write these requirements in https://autowarefoundation.github.io/autoware-documentation/main/installation and refer to the page.

Related: https://github.com/autowarefoundation/autoware/discussions/281

@LalithVipulananthan What do you think about it?

File: docs/tutorials/scenario-simulation/.pages
Comment (WJaworskiRobotec) [reply docs/tutorials/scenario-simulation/.pages:5]: so you mean to have docs/tutorials/scenario-simulation/planning-simulation/(scenario/random)-test-simulation.md ?

Comment (kenji-miyake) [reply docs/tutorials/scenario-simulation/.pages:5]: Exactly. It's because we'll add scenario testing with the digital-twin simulator in the future.

File: docs/tutorials/scenario-simulation/installation.md
Comment (kenji-miyake) [inline docs/tutorials/scenario-simulation/installation.md:11]: To reduce maintenance cost,
How about referring to the [installation page](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/) and writing only the diff on this page?

Comment (LalithVipulananthan) [reply docs/tutorials/scenario-simulation/installation.md:11]: I was about to make this exact same comment. Since Autoware needs to be built anyway, the only remaining tasks would be to import the simulator.repos file and then run colcon build to build the simulator specific components right?

File: docs/tutorials/scenario-simulation/random-test-simulation.md
Comment (LalithVipulananthan) [inline docs/tutorials/scenario-simulation/random-test-simulation.md:3]:
```suggestion
!!! note

    Running the Scenario Simulator requires some additional steps on top of building and installing Autoware, so make sure that [Scenario Simulator installation](installation.md) has been completed first before proceeding.
```

File: docs/tutorials/scenario-simulation/scenario-test-simulation.md
Comment (LalithVipulananthan) [inline docs/tutorials/scenario-simulation/scenario-test-simulation.md:3]: Same reasoning as for the random test runner instructions.

```suggestion
!!! note

    Running the Scenario Simulator requires some additional steps on top of building and installing Autoware, so make sure that [Scenario Simulator installation](installation.md) has been completed first before proceeding.
```

File: docs/tutorials/scenario-simulation/random-test-simulation.md
Comment (LalithVipulananthan) [inline docs/tutorials/scenario-simulation/random-test-simulation.md:7]: Slight rewording suggestion and typo correction.

```suggestion
1. Move to the workspace directory where Autoware and the Scenario Simulator have been built.
```

File: docs/tutorials/scenario-simulation/scenario-test-simulation.md
Comment (LalithVipulananthan) [inline docs/tutorials/scenario-simulation/scenario-test-simulation.md:7]: Slight rewording suggestion and typo correction.

```suggestion
1. Move to the workspace directory where Autoware and the Scenario Simulator have been built.
```

File: docs/tutorials/scenario-simulation/installation.md
Comment (kenji-miyake) [reply docs/tutorials/scenario-simulation/installation.md:11]: We need to run `rosdep install` again.

File: docs/tutorials/scenario-simulation/random-test-simulation.md
Comment (LalithVipulananthan) [inline docs/tutorials/scenario-simulation/random-test-simulation.md:26]: Slightly rewording and changed the link to go directly to the section about random test runner parameters.

```suggestion
For more information about supported parameters, refer to the ![random_test_runner documentation].(https://github.com/tier4/scenario_simulator_v2/blob/master/test_runner/random_test_runner/Readme.md#node-parameters)
```

File: docs/tutorials/scenario-simulation/installation.md
Comment (LalithVipulananthan) [reply docs/tutorials/scenario-simulation/installation.md:9]: I agree. Something like this could be written here instead:

> ## Prerequisites 
>
> 1. [A machine that supports the minimum requirements to run Autoware](../installation)
> 2. [Autoware has been built and installed](../installation/#installing-autoware)

File: docs/tutorials/scenario-simulation/installation.md
Comment (LalithVipulananthan) [reply docs/tutorials/scenario-simulation/installation.md:11]: Ah okay. That still reduces the installations steps down to three which is pretty short!

**Comment by WJaworskiRobotec** [comment]
@LalithVipulananthan @kenji-miyake I believe I've addressed all your comments. Thank you very much for good comments.

File: docs/tutorials/scenario-simulation/.pages
Comment (kenji-miyake) [reply docs/tutorials/scenario-simulation/.pages]: Since it was a bit incorrect, I've fixed `.pages`. :bow:  e322fb533b37e9fd7877c0a3eb0884dc3aafd4fa

Comment (WJaworskiRobotec) [reply docs/tutorials/scenario-simulation/.pages]: Thanks a lot !

File: docs/tutorials/scenario-simulation/planning-simulation/installation.md
Comment (kenji-miyake) [inline docs/tutorials/scenario-simulation/planning-simulation/installation.md:34]: [nits] The final characters are not consistent. Is that intended?

image

File: docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md
Comment (kenji-miyake) [inline docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md:26]: I've moved the directory.

File: docs/tutorials/scenario-simulation/planning-simulation/scenario-test-simulation.md
Comment (kenji-miyake) [inline docs/tutorials/scenario-simulation/planning-simulation/scenario-test-simulation.md:17]: Same here. https://github.com/autowarefoundation/autoware-documentation/pull/118/files#r879512508

File: docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md
Comment (kenji-miyake) [reply docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md:17]: As I'm not sure, I'll leave it to you!

File: docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md
Comment (kenji-miyake) [inline docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md:23]: For me, this is easier to view. If there are some reasons, the current format is okay.

```suggestion
   ros2 launch random_test_runner random_test.launch.py \
     architecture_type:=awf/universe \
     sensor_model:=sample_sensor_kit \
     vehicle_model:=sample_vehicle
```

File: docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md
Comment (WJaworskiRobotec) [reply docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md]: Ok i changed both to "Run the simulation", might be a bit more specific. good point

File: docs/tutorials/scenario-simulation/planning-simulation/scenario-test-simulation.md
Comment (kenji-miyake) [inline docs/tutorials/scenario-simulation/planning-simulation/scenario-test-simulation.md:25]: Same here. https://github.com/autowarefoundation/autoware-documentation/pull/118/files#r879522159
I'll leave it to you.

```suggestion
   ros2 launch scenario_test_runner scenario_test_runner.launch.py \
     architecture_type:=awf/universe \
     record:=false \
     scenario:='$(find-pkg-share scenario_test_runner)/scenario/sample.yaml' \
     sensor_model:=sample_sensor_kit \
     vehicle_model:=sample_vehicle
```

**Review by kenji-miyake — DISMISSED** [review]
Thank you for addressing my reviews! :pray:

**Comment by WJaworskiRobotec** [comment]
@kenji-miyake sorry, i had to psuh onve again to fix DCO, so i believe you need to approve once again

**Review by kenji-miyake — APPROVED** [review]
FYI, you can manually pass the DCO check.
https://probot.github.io/apps/dco/

File: docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md
Comment (kenji-miyake) [inline docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md:28]: @WJaworskiRobotec cc @HansRobo Could you update this link? It's 404 now.
https://github.com/autowarefoundation/autoware-documentation/runs/6769693334?check_suite_focus=true#step:3:581

Comment (HansRobo) [reply docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md:28]: Thank you for your kind notification!
I updated the link.
```suggestion
For more information about supported parameters, refer to the ![random_test_runner documentation].(<https://github.com/tier4/scenario_simulator_v2/blob/master/docs/user_guide/random_test_runner/README.md#node-parameters>)
```

Comment (kenji-miyake) [reply docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md:28]: @HansRobo Could you open a PR for that? :pray:

Comment (kenji-miyake) [reply docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md:28]: Or should I do it?

Comment (HansRobo) [reply docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md:28]: Oh, This PR has already merged!
I will create PR for this change now...

**Pull request merged**
